spec/services/exports/export_distributions_csv_service_spec.rb

```diff
@@ -65,7 +67,7 @@
         "Partner",
         "Date of Distribution",
         "Source Inventory",
-        "Total Items",
+        "Total number of Items",
```

Comment:
I there a way to conditionally change the header from "Total Items Distributed" to something like "Total #{item} Distributed" depending if an item is selected? I can forsee this being a little confusing.

@cielf what do you think?

Reply:
@edwinthinks I was talking to Scott during the office hours,  and there's an argument against that.    If the headers change on the csv that's exported, it might be a problem for people who are using the export for other reasons if it's changing all the time.

Reply:
Could we have two seperate columns then that show total items and total items of the filtered item?

Reply:
just trying to understand more, when exporting if there is no filter applied on the items, then what would the total items of filtered items would show?

Reply:
I think it wouldn't be included @kannans5 IMO